package.json

```diff
@@ -24,6 +24,10 @@
             "flowforge-nr-tools": "dist/plugin.js"
         }
     },
+    "keywords": [
+        "flowforge",
+        "node-red"
```

Comment:
Is this node to be published in the flows catalog Nick?

Reply:
Probably. Reserving the right to do so at this stage, but without the keyword, it wouldn't be an option.